client/components/domains/transfer-domain-step/index.jsx

```diff
@@ -114,21 +115,23 @@ class TransferDomainStep extends React.Component {
 				{ this.notice() }
 				<form className="transfer-domain-step__form card" onSubmit={ this.handleFormSubmit }>
 					<div className="transfer-domain-step__domain-description">
-						<p>{ translate( 'Use your own domain for your WordPress.com site.' ) }</p>
-						<p>
+						<div className="transfer-domain-step__domain-heading">
+							{ translate( 'Use your own domain for your WordPress.com site.' ) }
+						</div>
+						<div>
 							{ translate(
-								'Enter the domain you want to transfer to WordPress.com and manage your domain and site' +
-									" all in one place. Domains purchased in the last 60 days can't be transferred. {{a}}Learn More{{/a}}",
+								'Enter the domain you want to transfer to WordPress.com so you can manage your domain and site' +
```

Comment:
Hi! I've found a possible matching string that has already been translated **17** times:
`translate( 'Enter the domain you want to transfer to WordPress.com and manage your domain and site all in one place. Domains purchased in the last 60 days can't be transferred. {{a}}Learn More{{/a}}' )` *ES Score: 10*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).